docs/api/payments.rst

```diff
@@ -1,37 +1,72 @@
+| amount                 | The amount of payment                                                           |
++------------------------+---------------------------------------------------------------------------------+
+| state                  | :doc:`State of the payment process </book/orders/payments>`                     |
++------------------------+---------------------------------------------------------------------------------+
+| _links[self]           | Link to itself                                                                  |
++------------------------+---------------------------------------------------------------------------------+
+| _links[payment-method] | Link to the related payment method                                              |
++------------------------+---------------------------------------------------------------------------------+
+| _links[order]          | Link to the related order                                                       |
++------------------------+---------------------------------------------------------------------------------+
+
+.. note::
+
+    Read more about :doc:`Payments in the component docs</components/Payment/models>`.
+
+Getting a Single Payment
+------------------------
 
-Index of all payments
----------------------
+To retrieve the details of a payment you will need to call the ``/api/v1/payments/{id}`` endpoint with the ``GET`` method.
 
-You can retrieve the full list payment by making the following request:
+Definition
+^^^^^^^^^^
 
 .. code-block:: text
 
-    GET /api/v1/payments/
+    GET /api/v1/payments/{id}
 
-Parameters
-^^^^^^^^^^
++---------------+----------------+--------------------------------------+
+| Parameter     | Parameter type | Description                          |
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| id            | url attribute  | Id of the requested payment          |
++---------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details of the payment method with ``id = 20`` use the below method:
+
+.. code-block:: bash
 
-page
-    Number of the page, by default = 1
-limit
-    Number of items to display per page
-criteria[channel] *(optional)*
-    The channel id
-criteria[number] *(optional)*
-    The order number
-criteria[billingAddress] *(optional)*
-    First or last name of the customer bill to address
-criteria[createdAtFrom] *(optional)*
-    Starting date
-criteria[createdAtTo] *(optional)*
-    End date
-
-Response
-^^^^^^^^
+     $ curl http://demo.sylius.org/api/v1/payments/20 \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
+
+    The *id = 20* is just an example. Your value can be different.
```

Comment:
Everywhere we have "The *20* id is just an example" or "The *potato* code is just an example"